Main.py

```diff
@@ -2,431 +2,368 @@
 
 import discord
 from discord.ext import commands
-import asyncio
-import sqlite3
-from tabulate import tabulate
-import logging
-import os, sys
+
 import datetime
+import logging
 import math
+import os
 import pytz
+import sqlite3
+import statistics
 import subprocess
+import sys
 
-DB_PATH = './Scoreboard.db'
+CMD_PREFIX = "%"
+CROSSWORD_ROLE = "crosswords"
+CROSSWORD_TIMEZONE = "America/New_York"
+DB_PATH = "./Scoreboard.db"
+DEVELOPER_ROLE = "idoneam"
 
-cmd_prefix = '%'
-bot = commands.Bot(command_prefix=cmd_prefix)
+bot = commands.Bot(command_prefix=CMD_PREFIX)
 
 # Logging configuration
 logger = logging.getLogger('discord')
 logger.setLevel(logging.ERROR)
-handler = logging.FileHandler(filename='discord.log', encoding='utf-8', mode = 'w')
+handler = logging.FileHandler(filename='discord.log', encoding='utf-8', mode='w')
 handler.setFormatter(logging.Formatter('%(asctime)s:%(levelname)s:%(name)s: %(message)s'))
 logger.addHandler(handler)
```

Comment:
Nothing ends up in discord.log, every error message is in the console

Reply:
19e11bd1e01522040608edc461f68e8d7edf8cc6